hack/test-system.sh

```diff
@@ -37,8 +37,12 @@ EOF
     export CONTAINERS_STORAGE_CONF=/etc/containers/storage.conf
 fi
 
-# Build skopeo, install into /usr/bin
-make PREFIX=/usr install
+# Print what binary is being used for tests
+if [[ -v SKOPEO_BINARY ]]; then
+    echo "Testing with $SKOPEO_BINARY ..."
+else
+    echo "Testing with $(git rev-parse --show-toplevel)/bin/skopeo ..."
+fi
```

Comment:
Couldn't this be simplified with:

```bash
SKOPEO_BINARY=${SKOPEO_BINARY:-$(git rev-parse --show-toplevel)/bin/skopeo ...}
echo "Testing with $SKOPEO_BINARY ..."
```

Reply:
yup, thanks!

Reply:
removed this script to run everything inline in Makefile. Marking resolved for now.